scripts/import_templates.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# This file is part of ap_verify_ci_dc2.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Script for copying templates appropriate for these fields.
+
+The datasets can be from any source.
+
+Example:
+$ python import_templates.py -t "u/me/DM-123456-template"
+imports goodSeeing templates from u/me/DM-123456-template in /repo/dc2 to
+templates/goodSeeing in this dataset's preloaded repo. See
+import_templates.sh -h for more options.
+"""
+
+import argparse
+import logging
+import os
+import sys
+import tempfile
+
+import lsst.log
+import lsst.skymap
+from lsst.daf.butler import Butler, CollectionType
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
+
+
+########################################
+# Command-line options
+
+def _make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-b", dest="src_dir", default="/sdf/group/rubin/repo/dc2",
+                        help="Repo to import from, defaults to '/sdf/group/rubin/repo/dc2'.")
+    parser.add_argument("-t", dest="src_collection", required=True,
+                        help="Template collection to import from.")
+    parser.add_argument("-w", dest="where",
+                        help="Query datasets dimension expression to restrict the templates \
+                              that will be imported from the source collection..")
```

Comment:
Extra period, and you can use string auto-concatenation.

```suggestion

                        help="Query datasets dimension expression to restrict the templates "
                              "that will be imported from the source collection.")
```